app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -95,17 +95,27 @@ class FollowUsers extends Component {
   render() {
     const { users, selectedUsers } = this.state;
     const { prev } = this.props;
+    let followingStatus;
+    if (selectedUsers < 2) {
+      followingStatus = "You're not following anyone";
+    } else {
+      const everyone =
+        selectedUsers.length === users.length ? ' (everyone)' : '';
+      const object = selectedUsers.length === 1 ? 'person' : 'people';
+      followingStatus = `You're following ${selectedUsers.length} ${object}${everyone}`;
+    }
+
     return (
       <div className="onboarding-main">
         <Navigation prev={prev} next={this.handleComplete} />
-        <div className="onboarding-content">
+        <div className="onboarding-content no-padding-bottom">
```

Comment:
```suggestion
        <div className="onboarding-content footer">
```
Instead of naming the class based on the behavior/logic it contains, I think it makes more sense to name it based on the actual element in that needs that style. IMO, I don't need to know that we have `no-padding-bottom` added as a class on this `div` when I look at the JSX being rendered; it could just be abstracted out into a `footer` class and when I need to know what styles the footer has, I could just figure that out from the class.

Btw, `footer` is just a suggestion, it might not be the right name 🙃

Reply:
The habit of a Ruby developer 😆 . I updated it to `toggle-bottom`.